Sources/App/Context/HomeContext.swift

```diff
@@ -18,4 +18,5 @@ struct HomeContext: Content {
     var silverSponsors: [Sponsor] = []
     var goldSponsors: [Sponsor] = []
     var dropInSessions: [DropInSession] = []
+    var schedule: [[Slot]] = []
```

Comment:
Do we still need slots in the context with schedule added?

Reply:
Removed